os_core/src/structures/driver.rs

```diff
@@ -0,0 +1,9 @@
+use super::kernel_information::KernelInformation;
+
+pub type Registrator = extern "C" fn(KernelInformation) -> Driver;
+
+#[derive(Clone, Copy)]
+#[repr(C)]
+pub struct Driver {
+    pub serial: [u8; 16],
```

Comment:
also where does the serial array come from that you defined in all the devices? Did you just make it up?

Reply:
I just made it up. We want the drivers to have some kind of signature id (should have named the property like that tbh).

Reply:
Okay, figured so much just wanted to make sure